Makefile

```diff
@@ -318,16 +324,23 @@ goimports:
 	@printf "Modified files: $(MODIFIED_FILES)\n"
 	@$(GOIMPORTS_BIN) -w $(filter %.go, $(MODIFIED_FILES))
 
-lint: $(GOLANGCI_LINT)
+lint-actions: $(ACTIONLINT)
+	@printf $(COLOR) "Linting GitHub actions..."
+	@$(ACTIONLINT)
+
+lint-code: $(GOLANGCI_LINT)
+	@printf $(COLOR) "Linting code..."
+	@$(GOLANGCI_LINT) run --verbose --timeout 10m --fix=$(GOLANGCI_LINT_FIX) --new-from-rev=$(GOLANGCI_LINT_BASE_REV) --config=.golangci.yml
+
+lint: lint-code lint-actions lint-api lint-protos
```

Comment:
Won't this run the deps first, and then the print below? Also, why not call `lint-api` and `lint-protos` here too?